api/src/tests/helpers/testData.ts

```diff
@@ -10,3 +11,41 @@ export const testAsset: AssetAttributes = {
   createdAt: new Date(),
   updatedAt: new Date(),
 };
+
+export const testPaginationAssets: PaginationResult<AssetAttributes> = {
+  lastPage: 1,
+  totalRecords: 3,
+  hasMorePages: false,
+  data: [
+    {
+      id: 1,
+      assetTag: "testAssetTag",
+      name: "testAsset",
+      serialNumber: "",
+      modelNumber: "",
+      nextAuditDate: new Date(),
+      createdAt: new Date(),
+      updatedAt: new Date(),
+    },
+    {
+      id: 2,
+      assetTag: "testAssetTag1",
+      name: "testAsset1",
+      serialNumber: "",
+      modelNumber: "",
+      nextAuditDate: new Date(),
+      createdAt: new Date(),
+      updatedAt: new Date(),
+    },
+    {
+      id: 3,
+      assetTag: "testAssetTag2",
+      name: "testAsset2",
```

Comment:
W: Have numbers within `name` and `assetTag` match up with `id`, as opposed to being offset by 1.

Reply:
Commit: 02dbeb7a0fa4367798ed906384e2bba6522c1763